### DISCUSSION TITLE
Certificado do fundamentos de C# e assinatura Premium

### DISCUSSION THREAD

**Discussion by BrunoFaria2021** [balta-io/forum#219] [Tira Dúvidas] [answered]
Fala André, estou a concluir o curso de fundamentos C#, pretendo assinar o premium com vocês gostei muito do curso ótima didática sem palavras. 

Eu pego certificado do fundamento c# ? 

e na assinatura premium como funciona esses certificados?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BrunoFaria2021, como vai?
Muito obrigado pelos feedbacks, fico feliz que esteja curtindo nosso conteúdo 💜

Todos os cursos tem certificado, desde os gratuitos até os pagos 🚀
